index.html

```diff
@@ -11,13 +11,14 @@
   <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+  <script defer src="index.js"></script>
 </head>
 <body>
-  <main>
+  <main data-main>
     <header class="flex">
       <a href="#" class="logo">My Logo</a>
```

Comment:
- [ ] Please eliminate the extra white space at the bottom of the mobile menu to match the given design.
![image](https://user-images.githubusercontent.com/62290545/230911154-c213991e-815a-43da-9e6f-6b81f539fcc4.png)